server/models/User.model.ts

```diff
@@ -71,9 +71,11 @@ class User extends Model<UserAttributes, UserCreationAttributes> {
   website: string;
 
   @CreatedAt
+  @Column({ field: "created_at" })
```

Comment:
Instead of manually overriding the names, I'd probably opt to use the [`underscored`](https://sequelize.org/docs/v6/other-topics/naming-strategies/#the-underscored-option) option when initializing Sequelize.